.github/workflows/local_tests.yml

```diff
@@ -22,12 +23,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        quarkus-version: ['2.2.3.Final', '2.4.0.Final']
-        mandrel-version: ['21.2.0.2-Final', '21.3.0.0-Final']
+        quarkus-version: ['1.11.7.Final', '2.2.3.Final', '2.4.0.Final']
```

Comment:
Are we still interested in Quarkus 1.11 and Mandrel 20.3?

Reply:
@zakkak Not for testing Q or M. This is for knowing that TS is still compatible with it. It is too soon to go and purge all ```if (QUARKUS_VERSION.startsWith("1.")) {...``` and thus remove the compatibility.